labelImg.py

```diff
@@ -16,6 +18,7 @@
     from PyQt5.QtGui import *
     from PyQt5.QtCore import *
     from PyQt5.QtWidgets import *
+    # from PyQt5.QtWebEngineWidgets import *
```

Comment:
Could we remove it instead of commenting out

Reply:
OK, it is removed.